notebook2/notebook/templates/notebook-state.html

```diff
@@ -0,0 +1,58 @@
+
+<div class='nb'>
+    {% set ready = notebook.svc_status == 'Running'
+                    and notebook.pod_status == 'Running'
+                    and notebook.condition.status == 'True'
+                    and notebook.condition.type == 'Ready'
+    %}
+
+    {% if ready %}
+        <i
+        class="material-icons ready-icon"
+        >
+        done
+        </i>
+    {% else %}
+        <div class="spinner" ></div>
+    {% endif %}
+    <a
+        target="_blank"
+        class='nb-state-container'
+        href={{ notebook['url'] }}
+    >
+        <b>{{notebook.name}}</b>
+        <span className="small">
+        Condition:
+
+            {% if notebook.condition is not defined %}
+                <b>Initializing</b>
+            {% elif notebook.condition.status != "True" %}
+                <b>{{"Waiting for " ~ notebook.condition.type}}</b>
+            {% else %}
+                <b>{{notebook.condition.type}}</b>
+            {% endif %}
+        </b>
+        </span>
+        <span className="small">
+        Pod: <b>{{notebook.pod_status}}</b>
+        </span>
+        <span className="small">
+        Pod Name: <b>{{notebook.pod_name}}</b>
+        </span>
+        <span className="small">Created on: {{notebook.creation_date}}</span>
+    </a>
+    <form action='/notebook/delete' method='POST'>
+        <button type='submit' class="material-icons nb-close">
+            close
+        </button>
+    </form>
+</div>
+{% if not ready %}
+    <script>
+        let protocol = location.protocol.replace("http", "ws")
+        let sock = new WebSocket(protocol + location.host + "/wait");
+        sock.onmessage = function (event) {
+            window.location.reload()
```

Comment:
Why reload this page? We receive an event iff the notebook is ready, we should should redirect to the notebook.